scripts/Packages/Package_DFN_QFN/size_definitions/dfn.yaml

```diff
@@ -181,6 +181,45 @@ MLF-6-1EP_1.6x1.6mm_P0.5mm_EP0.5x1.26mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-8-1EP_2x2mm_P0.5mm_EP0.8x1.2mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.onsemi.com/pub/Collateral/NB3N551-D.PDF (page 7)'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 2
+    tolerance: 0.1
```

Comment:
The body size is given as "BSC" which means it is a reference value with 0 tolerance (the body tolerances are included in the tolerances for other measurements)

```yaml
body_size_x:
  nominal: 2
  tolerance: 0.0
```